ansible/roles/prometheus/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ prometheus_services:
     dimensions: "{{ prometheus_server_dimensions }}"
     haproxy:
       prometheus_server:
-        enabled: "{{ enable_prometheus_server }}"
+        enabled: "{{ enable_prometheus | bool }}"
```

Comment:
Should be enable_prometheus_server

Reply:
This is what came from upstream though

Reply:
From commit 55b2eee95b54880ee0656bef1f771eddee7cd7d9

Reply:
The stackhpc branch has a backport of the prometheus server disable commit: https://github.com/stackhpc/kolla-ansible/commit/078b6f749c6b5092f18bd7bf35db79377694f036

Reply:
The latest tip of this branch has all the missing commits and uses enable_prometheus_server